### PR TITLE
N23: wgpu section

### PR DESCRIPTION
**Pull request by kvark**
Part of #677

### CODE REVIEW

File: content/news/023/index.md [hunk 1]
```diff
@@ -110,6 +110,35 @@ projects. Pure Rust Vulkan allocators with support for erupt have been released.
...
+### [wgpu] family re-union
+
+![wgpu family reunion](wgpu-family-reunion.svg)
+
+[wgpu] is a [WebGPU] implementation in Rust. It is safe, efficient,
+and portable: can target both native and the Web.
+
+[Family reunion] is by far the biggest change in `wgpu` project since
+the inception. First, the Rust API of [wgpu-rs] was moved to the main
+[wgpu] repository. Whole code base was relicensed under MIT/Apache2.
```
Comment (cwfitzgerald) [inline content/news/023/index.md:122]:
```suggestion
[wgpu] repository. Second, the whole code base was relicensed under MIT/Apache2.
```

File: content/news/023/index.md [hunk 2]
```diff
@@ -110,6 +110,35 @@ projects. Pure Rust Vulkan allocators with support for erupt have been released.
 [erupt]: https://gitlab.com/Friz64/erupt
 [@Friz64]: https://blog.friz64.de/about
 
+### [wgpu] family re-union
+
+![wgpu family reunion](wgpu-family-reunion.svg)
+
+[wgpu] is a [WebGPU] implementation in Rust. It is safe, efficient,
+and portable: can target both native and the Web.
```
Comment (cwfitzgerald) [inline content/news/023/index.md:118]:
```suggestion
and portable: it can target both native and the Web.
```

File: content/news/023/index.md [hunk 3]
```diff
@@ -110,6 +110,35 @@ projects. Pure Rust Vulkan allocators with support for erupt have been released.
...
+
+[Family reunion] is by far the biggest change in `wgpu` project since
+the inception. First, the Rust API of [wgpu-rs] was moved to the main
+[wgpu] repository. Whole code base was relicensed under MIT/Apache2.
+
+[gfx-hal] - the Vulkan Portability-like graphics API abstraction -
+was detached from the project. Instead `wgpu` got its own in-house unsafe
+abstraction called "wgpu-hal" developed within [wgpu] repository.
```
Comment (cwfitzgerald) [inline content/news/023/index.md:126]:
```suggestion
abstraction called "wgpu-hal" developed within the [wgpu] repository.
```

File: content/news/023/index.md [hunk 4]
```diff
@@ -110,6 +110,35 @@ projects. Pure Rust Vulkan allocators with support for erupt have been released.
...
+
+The team released [wgpu-0.9] right before this transition, to give the new
+graphics infrastructure more time to take shape. At the time of writing,
+supported backends included Vulkan, Metal, and OpenGL ES3.
```
Comment (cwfitzgerald) [inline content/news/023/index.md:130]:
```suggestion
supported backends on the new hal include Vulkan, Metal, and OpenGL ES3.
```

File: content/news/023/index.md [hunk 5]
```diff
@@ -110,6 +110,35 @@ projects. Pure Rust Vulkan allocators with support for erupt have been released.
...
+
+Finally, the testing infrastructure received a major ugrade. It started
+rendering the examples on the available adapters and compraring the results
+with reference images.
```
Comment (cwfitzgerald) [inline content/news/023/index.md:134]:
```suggestion
with reference images. This includes automatic testing using software adapters on CI.
```

**Review by cwfitzgerald — COMMENTED** [review]
Some grammar and one addition

**Review by cwfitzgerald — APPROVED** [review]

File: content/news/023/index.md [hunk 6]
```diff
@@ -110,6 +110,36 @@ projects. Pure Rust Vulkan allocators with support for erupt have been released.
...
+
+[gfx-hal] - the Vulkan Portability-like graphics API abstraction -
+was detached from the project. Instead `wgpu` got its own in-house unsafe
+abstraction called "wgpu-hal" developed within the [wgpu] repository.
+
+The team released [wgpu-0.9] right before this transition, to give the new
+graphics infrastructure more time to take shape. At the time of writing,
+supported backends on the new HAL include Vulkan, Metal, and OpenGL ES3.
+
+Finally, the testing infrastructure received a major ugrade. It started
```
Comment (17cupsofcoffee) [inline content/news/023/index.md:132]:
```suggestion
Finally, the testing infrastructure received a major upgrade. It started
```

**Review by 17cupsofcoffee — APPROVED** [review]
lgtm :+1:

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**